build_tools/cmake/iree_bytecode_module.cmake

```diff
@@ -96,11 +96,30 @@ function(iree_bytecode_module)
     get_filename_component(_FRIENDLY_NAME "${_RULE_SRC}" NAME)
   endif()
 
+  # Check LLVM static library setting.
+  foreach(_FLAG ${_RULE_FLAGS})
+    string(FIND
+      "${_FLAG}" "--iree-llvm-static-library-output-path=" _STATIC_OBJ_IDX
```

Comment:
This flag still probably works with a single `-`, but this is specifically checking for `--`. Maybe just search for / replace `iree-llvm-static-library-output-path=`, with no dashes?

Reply:
Good catch! Modify the logic in the new commit to handle `-` and `--`, thanks!